packages/core-sdk/src/utils/chain.ts

```diff
@@ -42,7 +42,7 @@ export const mainnet = defineChain({
   contracts: {
     multicall3: {
       address: "0xca11bde05977b3631167028862be2a173976ca11",
-      blockCreated: 5882,
+      blockCreated: 340998,
```

Comment:
do you know what does this? what's the consequence of this not being the correct block before?